web-stories.php

```diff
@@ -60,15 +65,141 @@
 	define( 'WEBSTORIES_DEV_MODE', false );
 }
 
+// Load Compatibility class the old fashioned way.
+if ( ! class_exists( 'Web_Stories_Compatibility' ) ) {
```

Comment:
No need for this check? How would it ever already exist?

Reply:
Just being overly careful here.